modules/database-commons/src/test/java/org/testcontainers/ext/ScriptScannerTest.java

```diff
@@ -0,0 +1,49 @@
+package org.testcontainers.ext;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Test;
+
+import java.util.regex.Pattern;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ScriptScannerTest {
+    @Test
+    public void testHugeStringLiteral() {
+        String script = "/* a comment */    \"" +  StringUtils.repeat('~', 10000) + "\";";
```

Comment:
This line makes checkstyle fail. @eddumelendez do you know how to mark it as an exception?

Reply:
it was a double space after first `+`